index.html

```diff
@@ -65,6 +65,7 @@ <h2>Participants</h2>
         <a target="_blank" href="participants/michael-winter/">Michael Winter</a>
         <a target="_blank" href="participants/levermanx/index.html">M. X. Lever</a>
         <a target="_blank" href="participants/anthony-kusuma/index.html">Anthony Kusuma</a>
+		<a href="participants/yael-steinberger/yael-steinberger.html">Yael Steinberger</a>
```

Comment:
Unindent this please.

Reply:
Please add `target="_blank"` to the `<a>` tag as well so it will open in a new tab when clicked.